applications/SwimmingDEMApplication/custom_processes/transient_spatial_dependant_porosity_solution_body_force_process.cpp

```diff
@@ -190,8 +193,16 @@ void TransientSpatialDependantPorositySolutionBodyForceProcess::SetInitialBodyFo
         double& r_u1 = it_node->FastGetSolutionStepValue(EXACT_VELOCITY_X);
         double& r_u2 = it_node->FastGetSolutionStepValue(EXACT_VELOCITY_Y);
 
+        Matrix& permeability = it_node->FastGetSolutionStepValue(PERMEABILITY);
+
         if (std::pow(c * (x1 - x10), 2) + std::pow(((x2 - x20) / c), 2) < std::pow(R, 2)){
```

Comment:
This could be wrapped up in a function: `IsInsideEllipticalSupport()`. Otherwise perhaps a comment would help